l10n_it_fatturapa/bindings/README.md

```diff
@@ -0,0 +1,34 @@
+# Informazioni riguardanti la fatturazione elettronica
+
+## Incongruenze rispetto alle specifiche del file xsd
+
+* StringNNType: nelle specifiche da 1 a NN, lo SDI accetta da 0 a N
+* Amount8DecimalType, Amount2DecimalType, RateType, QuantitaType sono xs:decimal
+  ma la specifica vieta gli zeri finali, quindi li modifichiamo in xs:string
+
+## Istruzioni per generazione nuovi bindings
+
+* Scaricare il relativo xsd in locale
+* Applicare la patch allegata, adattarla se necessario
+* Generare i bindings utilizzando: `pyxbgen schema.xsd`
+* Rimuovere i riferimenti locali dai file py con il comando (adattando l'url se differente):
+
+  `sed -i -e "s,$(pwd),https://www\.fatturapa\.gov\.it/export/fatturazione/sdi/fatturapa/v1\.2\.1,g" *.py`
+
+* rinominare `binding.py` in `fatturapa_v_1_2.py`
+* riportare il prologo dai file `fatturapa_v_1_2.py` e `_ds.py` precedenti,
+  in particolare da `# flake8: noqa` a fine import, software come `meld` o similari
+  possono aiutare
+* sostituire i files precedenti
+* nel file `fatturapa_v_1_2` individuare la classe `DataFatturaType` ed aggiungere il seguente metodo:
+
+```python
+    # remove tzinfo from parsed dates or pyxb will fail the comparison
+    def __new__(cls, *args, **kwargs):
+        result = super(DataFatturaType, cls).__new__(cls, *args, **kwargs)
+        return result.replace(tzinfo=None)
+```
+
+La patch dell'xsd modifica il range di caratteri delle stringhe portandolo da 1-N a 0-N, inoltre aggiungere il tipo documento TD20 (Autofattura).
```

Comment:
Fa anche il cambio di alcuni campi da `decimal` a `string`

Reply:
verifico se `fractionMinDigits` ci piace altrimenti lo aggiungo

Reply:
Ho notato che nel caso in cui si spediscano fatture con destinatario estero, spesso capita di avere caratteri non Latin1 o con accenti nei nomi delle città, vie etc etc. Inoltre ho che ad esempio se scrivo come città `L'Aquila` non lo accetta, mentre se scrivo `L' Aquila` allora passa la validazione.

Reply:
@robyf70 non è che l'apostrofo originale è questo carattere qui? https://www.fileformat.info/info/unicode/char/2019/index.htm non è latin1 o extended latin1

se il nome del destinatario contiene un carattere non latin1 o extended latin1 ti attacchi, ringrazia chi ha fatto la specifica :smiley:

Reply:
@SimoRubi purtroppo nulla, pyxb non supporta `fractionMinDigits` quindi ci tocca usare stringa, ho aggiornato il readme e minimizzato le modifiche ai file generati mettendo il controllo in `__init__`